From 6915b8ddc149601e6f0baf4836f0f9f18e3ee25f Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 24 Jul 2019 20:22:57 +0300
Subject: [PATCH] arm64: dts: lx2160a: Update phy mode for Aquantia PHYs

The Aquantia driver does not allow xgmii mode anymore for
the AQR107 PHYs. Use the correct usxgmii mode instead.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -220,12 +220,12 @@
 
 &dpmac3 {
 	phy-handle = <&aquantia_phy1>;
-	phy-connection-type = "xgmii";
+	phy-connection-type = "usxgmii";
 };
 
 &dpmac4 {
 	phy-handle = <&aquantia_phy2>;
-	phy-connection-type = "xgmii";
+	phy-connection-type = "usxgmii";
 };
 
 &dpmac5 {
